tools/dm/shard-merge-scenario.md

```diff
@@ -0,0 +1,273 @@
+---
+title: DM 分片合并场景
+summary: 学习分片合并场景下如何使用 DM 同步数据
+category: tools
+---
+
+# Data Migration 分片合并场景
+
+本文介绍如何在分片合并场景中使用 Data Migration (DM)。使用场景中，三个上游 MySQL 实例的分库和分表数据需要同步至下游 TiDB 集群。
+
+## 上游实例
+
+假设上游库结构如下：
+
+- 实例 1
+
+    | 库 | 表 |
```

Comment:
```suggestion
    | Schema | Tables |
```

Reply:
We just keep the English version here. Please also update other similar terms in this PR. Thanks!